lib/CMakeLists.txt

```diff
@@ -48,6 +48,6 @@ add_dependencies(clspv_core clspv_c_strings clspv_glsl)
 
 if (MSVC)
   set_source_files_properties("${CMAKE_CURRENT_SOURCE_DIR}/SPIRVProducerPass.cpp"
-    PROPERTIES COMPILE_FLAGS "/Wall /WX /wd4710 /wd4820 /wd4625 /wd4626 /wd5026 /wd5027 /wd4061 /wd4711 /wd4996"
+    PROPERTIES COMPILE_FLAGS "/Wall /WX /wd4710 /wd4820 /wd4625 /wd4626 /wd5026 /wd5027 /wd4061 /wd4711 /wd4996 /wd4530 /wd4577"
   )
```

Comment:
What's 4577?  I didn't see it on msdn

https://msdn.microsoft.com/en-us/library/mt694070.aspx


The closest I could find was for HLSL : 
https://msdn.microsoft.com/en-us/library/windows/desktop/dn508414(v=vs.85).aspx